workflow/scripts/featurecounts_deseq2.R

```diff
@@ -317,11 +331,12 @@ if (file.exists(ResultsFile) == FALSE) {
             cat("\n",file=LogFile,append=TRUE,sep="")
         # AVI: creates required output files with message
         } else {
-            stop("More than 2 samples treated with the same antibody are needed to calculate the FDR & LOGFC.")
+            warning("More than 2 samples treated with the same antibody are needed to calculate the FDR & LOGFC.")
         }
 
         ## SAMPLE CORRELATION HEATMAP
-        pdf(file=snakemake@output[["plot_sample_corr_heatmap"]],width=10,height=8)  # AVI: added to create separate pdf files
+        CorrHeatmapFile <- file.path(snakemake@output[["plot_sample_corr_heatmap"]], paste(snakemake@wildcards[["antibody"]], ".", CompPrefix, ".correlation_heatmap.pdf", sep=""))  # AVI: dynamically file name extensions added
```

Comment:
```suggestion
        CorrHeatmapFile <- file.path(
            snakemake@output[["plot_sample_corr_heatmap"]],
            paste(snakemake@wildcards[["antibody"]], ".", CompPrefix, ".correlation_heatmap.pdf", sep="")
        )  # AVI: dynamically create file names
```
Improve readability.